lib/src/input/sketch/entities/style/border.dart

```diff
@@ -37,10 +37,10 @@ class Border {
 
   PBDLBorder interpretBorder() {
     return PBDLBorder(
```

Comment:
For these classes where we return an empty PBDLBorder, BorderOptions, Fill, etc. are these classes still necessary, or is this info. coming from somewhere else? 😮

Reply:
The problem was that this is sketch, & the information will come arranged differently, so until we support sketch again, I commented out this code